DayrSuryan/DSEthiop12.xml

```diff
@@ -57,7 +57,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                   <msItem xml:id="ms_i2">
                      <locus from="24r" to="37v"/>
-                     <title type="incomplete" ref="LIT4960Taabbeyo">The initial part of the prayer, with
+                     <title type="complete" ref="LIT4960Taabbeyo">Excerpt from the initial part of the prayer, with
```

Comment:
I think that writing this information inside `<title>` might lead to problems, Maybe it would be better to have a `<note>`.

Reply:
ok

Reply:
ok